packages/preview-middleware/src/base/flp.ts

```diff
@@ -79,7 +79,7 @@ const DEFAULT_THEME = 'sap_horizon';
 /**
  * Default path for mounting the local FLP.
  */
-const DEFAULT_PATH = '/test/flp.html';
+const DEFAULT_PATH = '/test/flpSandbox.html';
```

Comment:
Ehm ... no. It might be the default when wrapped by the `@sap/ux-ui5-tooling` but the default in the open-ux-tools has been `/test/flp.html` from the beginning, so changing this is quite a braking change (for no reason).

Reply:
OK, will revert the change 👍

Reply:
Done [ee99bad](https://github.com/SAP/open-ux-tools/pull/1930/commits/ee99bada8a7b30cd34a047ea9e91d66cbc661f10)